sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/build.sbt

```diff
@@ -0,0 +1,7 @@
+
+enablePlugins(JavaAgent)
+enablePlugins(AkkaGrpcPlugin)
+
+javaAgents += "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % "2.0.6" % "runtime"
```

Comment:
2.0.7 is already out

Reply:
copy-pasta, I can upgrade across the project if it matters